packages/block-library/src/image/edit.native.js

```diff
@@ -9,67 +9,104 @@ import RNReactNativeGutenbergBridge from 'react-native-gutenberg-bridge';
  */
 import { MediaPlaceholder, RichText, BlockControls } from '@wordpress/editor';
 import { Toolbar, ToolbarButton } from '@wordpress/components';
+import { Component } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import ImageSize from './image-size';
 
-export default function ImageEdit( props ) {
-	const { attributes, isSelected, setAttributes } = props;
-	const { url, caption } = attributes;
+class ImageEdit extends Component {
+	constructor() {
+		super( ...arguments );
+		this.onMediaLibraryPress = this.onMediaLibraryPress.bind( this );
```

Comment:
Hey @Tug ! I had to add this bind back since it was giving an error accessing `this.props.setAttributes`.

Reply:
Yep, usually `onSomething` methods need to be bound to `this` 👍